lti_consumer/lti_xblock.py

```diff
@@ -700,6 +711,10 @@ def lti_provider_key_secret(self):
         """
         Obtains client_key and client_secret credentials from current course.
         """
+        if self.lti_version == "external":
```

Comment:
Why do you need the secret here?

Reply:
Not sure exactly. I just added this block to maintain consistency with the previous logic of returning the key and secret. The only place I see this function `lti_provider_key_secret` seems to in a debug call.

Reply:
`lti_provider_key_secret` is only used when the configuration is set to `CONFIG_ON_XBLOCK`, [here](https://github.com/openedx/xblock-lti-consumer/blob/master/lti_consumer/models.py#L238) so there's no to modify this (the `LtiConfiguration` model ensures that the `LtiConsumer` client has the consistent configuration, so it's his job to determine where to fetch the credentials from).